bigml/sourcehandler.py

```diff
@@ -86,6 +88,21 @@ def _create_remote_source(self, url, args=None):
         body = json.dumps(create_args)
         return self._create(self.source_url, body)
 
+    def _create_inline_source(self, src_obj, args=None):
+        """Create source from inline data
+
+        """
+        create_args = {}
+        if args is not None:
+            create_args.update(args)
+        if 'source_parser' in create_args:
+            create_args['source_parser'] = json.dumps(
+                create_args['source_parser'])
+
+        create_args.update({"data": json.dumps(src_obj)})
```

Comment:
Now I see, you pass the list and use dumps to turn it into string. I looks like wintermute is able to parse this JSON string, so forget my previous comment :-)